nebula-components/components/_navbar.scss

```diff
@@ -7,6 +7,12 @@ $nb-navbar-toggle-color: $nb-color-black !default;
 $nb-navbar-toggle-background: $nb-color-white !default;
 $nb-navbar-collapsed-width: 350px !default;
 
+@if $nb-navbar-sticky {
+  .#{$nb-namespace}o-site-wrap--padding-top {
+    padding-top: $nb-navbar-height; // <= TODO
```

Comment:
I suggest that we create a `c-site-content` component that has this `if` statement within then we can avoid mutating the site-wrap object.